app/queries/gossip_node_query.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+class GossipNodeQuery
+  def initialize(network: "mainnet")
+    @network = network
+    @query = <<~SQL
```

Comment:
It's possible to write this query with using Active Record, so maybe we should use that.